modules/installation-initializing-manual.adoc

```diff
@@ -134,12 +134,21 @@ it in the `<installation_directory>`.
 You must name this configuration file `install-config.yaml`.
 ====
 ifdef::restricted[]
+
 ** Unless you use a registry that {op-system} trusts by default, such as
 `docker.io`, you must provide the contents of the certificate for your mirror
 repository in the `additionalTrustBundle` section. In most cases, you must
 provide the certificate for your mirror.
 ** You must include the `imageContentSources` section from the output of the command to
 mirror the repository.
+** The 'ImageContentSourcePolicy' is generated as an output of `oc-mirror` flow (after the mirroring process is finished).
```

Comment:
ImageContentSourcePolicy was [deprecated in 4.13][1].  Is that still what `oc-mirror` is generating in 4.14?

[1]: https://docs.openshift.com/container-platform/4.13/release_notes/ocp-4-13-release-notes.html#ocp-4-13-icsp

Reply:
Thanks for that input, @wking ! I'm using the information from the jira. Would it work if I indicated in a NOTE that ImageContentSourcePolicy is not available in 4.13+?

Reply:
I'm glad you chimed in, I need review of this PR so I can get it completed and closed out!

Reply:
@wking there's a section in the documentation that describes the issues with ImageContentSourcePolicy. I'm referring to that in the note. please PTAL!